components/03-modules/products-grid-item/_products-grid-item.scss

```diff
@@ -0,0 +1,194 @@
+$products-grid-item__action-icon-size          : 16px !default;
+$products-grid-item__action-icon-size--wishlist: 20px !default;
+$products-grid-item__action-item-size          : 32px !default;
+$products-grid-item__action-item-spacing       : $spacer !default;
+$products-grid-item__base-spacing              : $spacer !default;
+$products-grid-item__content-bg-color          : $white !default;
+$products-grid-item__padding                   : 0 !default;
+$products-grid-item__text-padding              : 0 $spacer !default;
+$products-grid-item__border                    : 1px solid $gray-lightest !default;
+$products-grid-item__font-size-medium          : $font-size-base !default;
+$products-grid-item__font-size-large           : 18px !default;
+$products-grid-item__image-margin-bottom       : $spacer !default;
+$products-grid-item__name-color                : $font-color-base !default;
+$products-grid-item__name-font-weight          : $font-weight-normal !default;
+$products-grid-item__price-base-color          : $bg-color-dark !default;
+
+.products-grid-item {
+    max-width: 320px; // for proper imageHeight calculation based on container width
+    display: block;
+    padding: $products-grid-item__padding;
+    border: $products-grid-item__border;
+    position: relative;
+    overflow: hidden;
+
+    &__link {
+        &--name {
+            flex-basis: 70%;
+        }
+        @include links(active) {
+            display: block;
+            text-decoration: none;
+        }
+    }
+
+    &__content {
+        background: $products-grid-item__content-bg-color;
+        transition: all 0.5s ease-in-out;
+    }
+
+    &__details {
+        display: flex;
+        justify-content: space-between;
+        flex-wrap: wrap;
```

Comment:
You need to check margins and paddings with the design, b/c it doesn't look the same.

Reply:
changed some margins/padding but have question here
https://app.zeplin.io/project/59d7824828a80b524cc9de18/screen/59d79a29c42ab4d2da91eb4c (comment on zeplin)